build/tasks/build.js

```diff
@@ -82,13 +89,15 @@ module.exports = function( grunt ) {
 			contents = contents
 				.replace( /define\(\[[^\]]*\]\)[\W\n]+$/, "" );
 		}
+
 		// AMD Name
-		if ( (amdName = grunt.option( "amd" )) != null && /^exports\/amd$/.test( name ) ) {
-			if (amdName) {
+		if ( ( amdName = grunt.option( "amd" ) ) != null && /^exports\/amd$/.test( name ) ) {
```

Comment:
For now, I'm okay with leaving it as-is. If we agree to change it, we'll do another PR.